samples/snippets/fsharp/VS_Snippets_CLR_System/system.convert.tostring2/fs/tostring2.fs

```diff
@@ -0,0 +1,172 @@
+open System
+
+let convertByte =
+//<snippet9>
+    let bases = [2; 8; 10; 16]
+    let (numbers:byte list) = [ Byte.MinValue; 12uy; 103uy; Byte.MaxValue ]
```

Comment:
Just wanted to be explicit but you are right, where type inference works it's better to omit that.